### ISSUE TITLE
ReferenceError: MongoInternals is not defined

### ISSUE THREAD

**Issue by JoeyAndres** [xolvio/cleaner#13]
The following error occurs when **package** testing:

``` bash
W20160524-15:04:50.610(-6)? (STDERR) ReferenceError: MongoInternals is not defined
W20160524-15:04:50.610(-6)? (STDERR)     at packages/xolvio:cleaner/cleaner.js:36:15
```

I have figured out the solution. Basically, in the case of package testing, explicitly add 'mongo' (and aswell as underscore) as dependencies. I'll make a pr.

**Comment by JoeyAndres** [comment]
https://github.com/xolvio/cleaner/pull/14

**Comment by ghost** [comment]
Fix has been released as 0.3.1
